react-components/src/components/CacheProvider/FdmNodeCache.ts

```diff
@@ -114,43 +112,41 @@ export class FdmNodeCache {
   }
 
   private async getNonCachedModelMappings(
-    uniqueIds: DmsUniqueIdentifier[],
+    instances: DmsUniqueIdentifier[],
     modelRevisions: ModelRevisionId[]
   ): Promise<ThreeDModelFdmMappings[]> {
-    if (modelRevisions.length === 0 || uniqueIds.length === 0) {
+    if (modelRevisions.length === 0 || instances.length === 0) {
       return [];
     }
 
-    const fdmKeySet = new Set(uniqueIds.map((id) => createFdmKey(id.space, id.externalId)));
+    const fdmKeySet = new Set(instances.map((id) => createFdmKey(id.space, id.externalId)));
 
     const revisionToEdgesMap = await this.getAndCacheRevisionToEdgesMap(modelRevisions, false);
 
-    const modelDataPromises = modelRevisions.map(async ({ modelId, revisionId }) => {
+    return modelRevisions.map(({ modelId, revisionId }) => {
       const revisionKey = createModelRevisionKey(modelId, revisionId);
-      const edges = revisionToEdgesMap.get(revisionKey);
+      const connections = revisionToEdgesMap.get(revisionKey);
```

Comment:
Maybe it is better to change the `revisionToEdgesMap` to something like `revisionToConnectionsMap` as well